roles/network_plugin/contiv/templates/contiv-etcd.yml.j2

```diff
@@ -0,0 +1,41 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: contiv-etcd
+  namespace: {{ system_namespace }}
+  labels:
+    k8s-app: contiv-etcd
+spec:
+  selector:
+    matchLabels:
+      k8s-app: contiv-etcd
+  template:
+    metadata:
+      labels:
+        k8s-app: contiv-etcd
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      hostPID: true
```

Comment:
Since the api is trying to access etcd at 127.0.0.1:6666 with host networking, you need to ensure this pod runs only on masters. Add this block:

```
      nodeSelector:
        node-role.kubernetes.io/master: "true"
      tolerations:
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
```

Reply:
it requires to be run on both nodes, the ones on masters are member of cluster, on worker nodes are in proxy mode